source/common/upstream/maglev_lb.h

```diff
@@ -81,14 +82,15 @@ class MaglevLoadBalancer : public ThreadAwareLoadBalancerBase {
   createLoadBalancer(const NormalizedHostWeightVector& normalized_host_weights,
                      double /* min_normalized_weight */, double max_normalized_weight) override {
     return std::make_shared<MaglevTable>(normalized_host_weights, max_normalized_weight,
-                                         table_size_, stats_);
+                                         table_size_, use_hostname_, stats_);
   }
 
   static MaglevLoadBalancerStats generateStats(Stats::Scope& scope);
 
   Stats::ScopePtr scope_;
   MaglevLoadBalancerStats stats_;
   const uint64_t table_size_;
+  const bool use_hostname_;
```

Comment:
Same comment in the code as the config. Can you make this variable more descriptive/longer based on what we decide the config name should be?